libraries/Microsoft.Bot.Builder/TokenResolver.cs

```diff
@@ -65,10 +67,11 @@ private static async Task PollForTokenAsync(BotFrameworkAdapter adapter, ILogger
             }
 
             var stopwatch = Stopwatch.StartNew();
+            var oauthClient = turnContext.TurnState.Get<OAuthClient>();
 
             while (stopwatch.Elapsed < pollingTimeout && !shouldEndPolling)
             {
-                tokenResponse = await adapter.GetUserTokenAsync(turnContext, connectionName, null, cancellationToken).ConfigureAwait(false);
+                tokenResponse = await adapter.GetUserTokenAsync(turnContext, oauthClient?.Credentials as AppCredentials, connectionName, null, cancellationToken).ConfigureAwait(false);
```

Comment:
What happens in the null case?
Is there any test case for it?

Reply:
Yeah there's a bunch of test changes in TestAdapterTests.cs that cover the new overloads. For the null cases, it's actually covered in existing tests because they don't pass in an additional set of credentials so in those cases it'll be null